PokemonCollection.java

```diff
@@ -100,46 +56,61 @@ public void loadData(String filename)
 		       **/
 		        while(line != null && !line.isEmpty())
 		        {
-		            String[] pokeData = line.split(",");
-	                addOrModifyPokemon(pokeData[0], pokeData[1], pokeData[2], pokeData[3], pokeData[4], 
-	                 		 			pokeData[5], pokeData[6], pokeData[7], pokeData[8], 
-		            		 			pokeData[9], pokeData[10]);			             
-		            line = br.readLine();
+		        	String[] pokeData = line.split(",");
+	               
+		        	// this will read in Identification Stats: ID, name, gender, type
+		        	Stats.IdentifyStats myIS = new Stats().new IdentifyStats(Integer.parseInt(pokeData[0]),
+		        															  pokeData[1],
```

Comment:
pokeData[1],
  pokeData[2],
  pokeData[3],
Integer.parseInt(pokeData[4]));
		        	
For some reason on here, it not showing these, but I have them on my file.